packages/python/accern_data/accern_data.py

```diff
@@ -292,56 +354,88 @@ def __init__(
             token: str) -> None:
         self._base_url = url
         self._token = token
-        self._filters = self.validate_filters(None)
+        self._filters = self.parse_filters(self.validate_filters(None))
```

Comment:
`= {}`?

Reply:
?

I'm saying that you don't need to call any functions since {} is always the valid default filter